labelbox/client.py

```diff
@@ -1574,6 +1574,30 @@ def get_catalog_slice(self, slice_id) -> CatalogSlice:
         res = self.execute(query_str, {'id': slice_id})
         return Entity.CatalogSlice(self, res['getSavedQuery'])
 
+    def unarchive_feature_schema_node(self, ontology_id: str,
+                                      root_feature_schema_id: str) -> bool:
+        """
+        Returns true if the root feature schema node was successfully unarchived, false otherwise
+        Args:
+            root_feature_schema_id (str): The ID of the root level feature schema
+            ontology_id (str): The ID of the ontology
+        Returns:
+            bool
+        """
+        ontology_endpoint = self.rest_endpoint + "/ontologies/" + urllib.parse.quote(
+            ontology_id) + '/feature-schemas/' + urllib.parse.quote(
+                root_feature_schema_id) + '/unarchive'
+        response = requests.patch(
+            ontology_endpoint,
+            headers=self.headers,
+        )
+        if response.status_code == requests.codes.ok:
+            return response.json()['unarchived']
```

Comment:
It's probably fine to return `None` here, since it'll just return true when successfully unarchived anyway. Unless there's a case where unarchived = false

Reply:
It'll return false if the user passes in a root feature schema id that does not belong to the ontology.

Reply:
If the response is false, maybe we can just raise an exception then with an error message indicating that the unarchive was unsuccessful? I know the docstring specifies the behavior, but customers may not necessarily check for the output of this function, so it may be safer to throw an explicit message indicating the behavior

Reply:
[Updated](https://github.com/Labelbox/labelbox-python/pull/939/commits/2c45ec7cf3362b941a2630c6ef6c1ffde7b2fc00)